configure.ac

```diff
@@ -1,6 +1,6 @@
 AC_PREREQ([2.68])
 
-AC_INIT([CIFSD tools], [exfat_tools_version],
+AC_INIT([exfatprogs], [exfatprogs_version],
 	[linkinjeon@gmail.com])
```

Comment:
Don't you change this email?
And we need to get exfatprogs version from include/version.h. I will handle it.

Reply:
Yep, This is the leftover from Luca's patch. You can remove it on another patch.